### PR TITLE
Create math-frexp.md

### PR DESCRIPTION
**Pull request by haruharu1017**
Description
My topic is math.frexp(). I used the topic entry template and rename it as the file needed then edit in that file.

Issue Solved
Closes https://github.com/Codecademy/docs/issues/5085

Type of Change
Adding a new entry
Checklist
[✅] All writings are my own.
[✅] My entry follows the Codecademy Docs style guide.
[✅] My changes generate no new warnings.
[✅] I have performed a self-review of my own writing and code.
[✅] I have checked my entry and corrected any misspellings.
[✅] I have made corresponding changes to the documentation if needed.
[✅] I have confirmed my changes are not being pushed from my forked main branch.
[✅] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
[✅] I have linked any issues that are relevant to this PR in the Issues Solved section.

### CODE REVIEW

**Comment by haruharu1017** [comment]
@cigar-galaxy82 I launched a new pr, please review it, thank you!

**Comment by haruharu1017** [comment]
@cigar-galaxy82 I noticed that my code failed the format:verify and lint:md checks, but I'm having trouble identifying the exact issues. Could you please help me understand what went wrong with these checks?

File: content/python/concepts/math-module/terms/math-frexp/math-frexp.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+_x_ must be a valid number, or the function will return `TypeError`.
+
+##Example
+
+Use `math.frexp()` to return the mantissa and exponent of 10: 
+
+```py
+import math
+
+print(math.frexp(10))
```
Comment (cigar-galaxy82) [inline content/python/concepts/math-module/terms/math-frexp/math-frexp.md:33]:
```suggestion
import math

mantissa, exponent = math.frexp(10)

print("Mantissa of 10: ", mantissa)
print("Exponent of 10: ", exponent)
```

File: content/python/concepts/math-module/terms/math-frexp/math-frexp.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+The above code gives the following output:
+
+```shell
+(0.625, 4)
```
Comment (cigar-galaxy82) [inline content/python/concepts/math-module/terms/math-frexp/math-frexp.md:39]:
```suggestion
Mantissa of 10:  0.625
Exponent of 10:  4
```

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
hi @haruharu1017 thank you for contributing to Codecademy docs, I have suggested some changes please modify them.

**Comment by cigar-galaxy82** [comment]
The error you were getting was because of extra space in some lines.

**Review by cigar-galaxy82 — APPROVED** [review]

**Comment by mamtawardhani** [comment]
Hey @haruharu1017, since this PR has been raised from the `main` branch, you will have to re-raise it. Make sure you create a feature branch first, then edit the content and make other changes on the feature branch itself. Please refer to the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide) to look through the steps.

Closing this PR, thank you! 😄

**Pull request closed without merging**